unitex/src/fr/umlv/unitex/utils/SingleInstanceMonitor.java

```diff
@@ -0,0 +1,248 @@
+package fr.umlv.unitex.utils;
+
```

Comment:
Avoid mixing spaces and tabs. Fix any inconsistent indentation using 2-spaces instead of tabs.